.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,44 @@
+name: Feature Request
+description: Submit a feature request
+#title: ""
+labels: ["enhancement"]
+#assignees:
+#  -
+body:
+  - type: checkboxes
+    attributes:
+      label: Is there an existing issue for this?
+      description: Please search to see if an issue already exists for this enhancement.
+      options:
+      - label: I have searched the existing issues
+        required: true
+  - type: dropdown
+    id: area
+    attributes:
+      label: Area
+      options:
+        - Plutus Application Framework - Related to the Plutus application backend (PAB), emulator, Plutus libraries
+        - Other - Any other topic (Playgrounds, etc.)
```

Comment:
Since the repo split only two are relevant. Do we even want this anymore?

Reply:
agree; i think the option can go entirely